generator/uscore/uscore_generator.rb

```diff
@@ -322,7 +345,8 @@ def create_references_resolved_test(sequence)
         test = {
           tests_that: 'All references can be resolved',
           index: sequence[:tests].length + 1,
-          link: 'https://www.hl7.org/fhir/DSTU2/references.html'
+          link: 'https://www.hl7.org/fhir/DSTU2/references.html',
```

Comment:
These tests shouldn't have any DSTU2 references.

Reply:
Updated to r4 links